feature-group-definitions/masonry.yml

```
@@ -1,11 +1,6 @@
 name: Masonry
+description: Masonry is a type of grid layout where the items on one of the axes are tightly packed together, instead of leaving gaps to align across the other axis.
```

A couple of ideas for this one:

- Note the domain here (styles)
- Explain the metaphor

```suggestion
description: Masonry is a type of CSS grid layout where the items on one of the axes are tightly packed together, like brickwork, instead of leaving gaps to align across the other axis.
```